scripts/install.sh

```
@@ -0,0 +1,61 @@
+
+log()  { printf '\033[1;36m== %s\033[0m\n' "$*"; }
+ok()   { printf '\033[1;32m%s\033[0m\n' "$*"; }
+warn() { printf '\033[1;33mwarning: %s\033[0m\n' "$*" >&2; }
+die()  { printf '\033[1;31merror: %s\033[0m\n' "$*" >&2; exit 1; }
+
+command -v python3 >/dev/null 2>&1 || die "python3 not found (need 3.11+). Install it with your package manager, then re-run."
+command -v curl >/dev/null 2>&1 || die "curl not found. Install curl, then re-run."
+command -v tar >/dev/null 2>&1 || die "tar not found. Install tar, then re-run."
+
+if [ ! -d "$HOME/wemod-launcher" ]; then
+  warn "~/wemod-launcher not found - installing wemod-launcher first."
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)